DataCapturing.podspec

```diff
@@ -25,7 +25,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'DataCapturing'
-  s.version          = '4.2.3'
+  s.version          = '4.3.0'
```

Comment:
new function to set the update interval for the event handler justifies increase in second place.